### ISSUE TITLE
can't build source file

### ISSUE THREAD

**Issue by MetroShonen** [Nickk888SAMP/TextDraw-Editor#5]
Downloaded the whole package and put it in my server folder, had to change ndialog-pages.inc's includes because it was including y_hooks wrong, then when I tried to compile it I got tons of errors and warnings (mostly const-correctness related)

`C:\Users\nonso\Desktop\SAMP server\filterscripts\ntd.pwn:838 (warning) literal array/string passed to a non-const parameter     
C:\Users\nonso\Desktop\SAMP server\filterscripts\ntd.pwn:838 (warning) literal array/string passed to a non-const parameter     
C:\Users\nonso\Desktop\SAMP server\filterscripts\ntd.pwn:838 (warning) literal array/string passed to a non-const parameter     `

There's also a few errors that won't let the file build 
`C:\Users\nonso\Desktop\SAMP server\dependencies\YSI-Includes\YSI_Core\y_core\y_utils_impl.inc:282 (error) function heading differs from prototype`

Because the code for OnPlayerDisconnect in your filterscript is 
`public OnPlayerDisconnect(playerid)`
instead of 
`public OnPlayerDisconnect(playerid, reason)`

Also 
```C:\Users\nonso\Desktop\SAMP server\filterscripts\ntd.pwn:998 (error) undefined symbol "cmd_ntd"
C:\Users\nonso\Desktop\SAMP server\filterscripts\ntd.pwn:1014 (error) undefined symbol "cmd_ntd"
C:\Users\nonso\Desktop\SAMP server\filterscripts\ntd.pwn:1392 (error) undefined symbol "cmd_ntd"
C:\Users\nonso\Desktop\SAMP server\filterscripts\ntd.pwn:1393 (error) undefined symbol "cmd_ntd"
C:\Users\nonso\Desktop\SAMP server\filterscripts\ntd.pwn:1403 (error) undefined symbol "cmd_ntd"```

**Comment by Nickk888SAMP** [comment]
I think you got the errors by changing the the include, i've no errors and warnings, try using Zeex's Compiler: [https://github.com/pawn-lang/compiler/releases](https://github.com/pawn-lang/compiler/releases)
Also try using my YSI package, the undefined symbol i thing is because of the ZCMD include, but i've no errors.. 

"function heading differs from prototype" is not an error but warning, it'll compile the file, i've fixed that, thanks for giving me the hint on how to fix that!

**Comment by ghost** [comment]
use YSI 5.x if you using newest version of pawn-lang compiler

**Comment by Nickk888SAMP** [comment]
Errors has been fixed in version 6.0, warnings came from the new version of Zeex Pawn compiler, i've "Const-Corrected" the code so it will now not show those warnings.
